docs/faq.md

```diff
@@ -176,9 +180,14 @@ In some cases, the Sigrid CI client script might produce an error message `[SSL:
 
 If your environment requires a certificate store in a custom location, the environment variables `openssl_cafile_env` and `openssl_capath_envz` can be used to point Python to the correct location. Refer to the [Python documentation on OpenSSL](https://docs.python.org/3/library/ssl.html) for more information
 
-### Why can't I use the publish and pathprefix options together?
+### I used the subsystem option and see results from other subsystems
 
-You can either use the `--publish` option to publish your code to Sigrid, or you can use the `--pathprefix` option to receive feedback on a specific part of your codebase, but you cannot use both in combination with each other. The reason is that these two options relate to different usage scenarios. `--pathprefix` is for pull request integration, and indicates you want to use Sigrid CI for a *subset of your codebase*. In contrast, `--publish` indicates you want to publish *your entire codebase* to Sigrid. This is why the combination doesn't really make sense, it would indicate you simultaneously consider your repository as the entire codebase (since you want to publish it) and a part of the codebase (since you're asking for more specific feedback). Please [contact us](mailto:support@softwareimprovementgroup.com) if you believe you have a need for using the combination of these two options.
+If your repository is part of a larger system you can use subsystem to only get feedback on your repository. This is
+done on a best effort basis and you might get unexpected results if another subsystem published to Sigrid while your
+pipeline was running. To analyze subsystems Sigrid CI uses both the source code and measurement results from the last
+published measurement. If something is published while we retrieve these discrepancies might occur. If this happens, try
```

Comment:
Maybe we can re-write more clearly as:

"If results are published while we retrieve these measurements, some discrepancies might occur."